### PR TITLE
Swaps out some old icons for some nicer icons.

### PR DESCRIPTION
**Pull request by TheDanseMacabre**
Potentially the first of a few PRs. The soda machine comes from Polaris, stools from Bay, and grilles from /tg/.

If there's any fuckery it's because I used an outdated fork of the repo to make this PR.

### CODE REVIEW

**Comment by TheDanseMacabre** [comment]
!review

**Comment by alsoandanswer** [comment]
no to the soda machine

**Comment by fernerr** [comment]
I'd much prefer the barstool be added as a separate object from the regular stool, as they can serve different purposes. The mapping in of barstools I can handle in a later pr, alternatively, if you send me the sprites I'll add the object and map them in as necessary myself.

**Comment by Geevies** [comment]
the new soda machine is good, the bar stool needs to be another variant, grille looks just about the same

**Comment by alsoandanswer** [comment]
you should probably ask around first

**Comment by NonQueueingMatt** [comment]
I really like all of them, personally, but do make a forum post.

File: html/changelogs/dansemacabre-vending.yml [hunk 1]
```diff
@@ -38,4 +38,4 @@ delete-after: True
 # Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
 # Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
 changes: 
-  - backend: "Fixed some backend voting problems."
+  - imageadd: "Replaces the sprites for the soda vending machine, stools, and grilles."
```
Comment (NonQueueingMatt) [inline html/changelogs/dansemacabre-vending.yml]: Interesting changelog choice. You should delete this, make a copy from the example.yml and edit that.

Comment (TheDanseMacabre) [reply html/changelogs/dansemacabre-vending.yml]: As I said, this was github being weird, not necessarily my fault. I'll see if I can fix it, though.

**Comment by TheDanseMacabre** [comment]
https://forums.aurorastation.org/topic/14034-danses-icon-swaps/

**Pull request merged**